front/app/containers/Admin/reporting/components/ReportBuilder/TopBar/index.tsx

```diff
@@ -212,19 +204,37 @@ const ContentBuilderTopBar = ({
             </Text>
           )}
         </Box>
-        <LocaleSwitcher
-          selectedLocale={selectedLocale}
-          onSelectLocale={setSelectedLocale}
-        />
-        <Box mx="20px">
-          <PrintReportButton reportId={reportId} />
+        <Box>
+          <LocaleSelect locale={selectedLocale} setLocale={setSelectedLocale} />
+        </Box>
+        {!!phaseId && (
+          <Box ml="32px">
+            <ViewPicker view={view} setView={setView} />
+          </Box>
+        )}
+        <Box ml="32px">
+          <Button
+            icon="print"
+            buttonStyle="secondary"
+            iconColor={colors.textPrimary}
+            iconSize="16px"
+            px="12px"
+            py="8px"
+            linkTo={`/admin/reporting/report-builder/${reportId}/print`}
+            openLinkInNewTab
+          />
         </Box>
         <SaveButton
           disabled={disableSave}
           processing={isLoading}
           bgColor={saved ? colors.success : undefined}
           icon={saved ? 'check' : undefined}
           onClick={handleSave}
+          fontSize="14px"
+          ml="8px"
+          px="12px"
+          pb="3px"
```

Comment:
```suggestion
          pb="4px"
```
Should we stick to numbers divisible by four?

Reply:
In general yes, but in this case no. Otherwise the button looks slightly larger than all the other ones.